### ISSUE TITLE
Fix Base.show() when using "SolutionSavingCallback" with save_times

### ISSUE THREAD

**Issue by svchb** [trixi-framework/TrixiParticles.jl#487]

**Comment by efaulhaber** [comment]
Works for me?
```
┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
│ StepsizeCallback                                                                                 │
│ ════════════════                                                                                 │
│ is constant: ……………………………………………… true                                                             │
│ CFL number: ………………………………………………… 0.9                                                              │
└──────────────────────────────────────────────────────────────────────────────────────────────────┘
```

**Comment by svchb** [comment]
copied the wrong line :D

**Comment by efaulhaber** [comment]
See #485.

**Comment by svchb** [comment]
merged #485
